### ISSUE TITLE
nuget package not found?

### ISSUE THREAD

**Issue by kmscode** [mkaring/ConfuserEx#29]
The readme notes:

> The nuget package can be accessed using the [AppVeyor Nuget Feed](https://ci.appveyor.com/nuget/confuserex-r4olq7m3uysu) or it has to be self-hosted.

But there doesn't seem to be any packages available from the feed:

```
C:\Users\user>nuget list -Source https://ci.appveyor.com/nuget/confuserex-r4olq7m3uysu
No packages found.
```

Thanks in advance!

**Comment by mkaring** [comment]
There was a mistake with the 1.1 release that caused that the nuget package for this release was not published to the nuget feed. How ever the current prerelease build is up to date. you can query it using:

```
nuget list -Source https://ci.appveyor.com/nuget/confuserex-r4olq7m3uysu -Prerelease -AllVersions
```

Please use the 1.2 pre release version for now until the release of 1.2. I'll fix the issue until then. The 1.2 pre release version is currently equal to the 1.1 release plus the fix for #25.

**Comment by kmscode** [comment]
Thanks for the super quick response!
